app/src/main/kotlin/com/wire/android/ui/calling/ongoing/participantsview/ParticipantTile.kt

```diff
@@ -227,6 +230,8 @@ private fun CameraPreview(
             }
         )
     } else {
+        if (isCameraStopped) return
+        isCameraStopped = true
         onClearSelfUserVideoPreview()
     }
```

Comment:
question: will it be better if we have this block inside a LaunchEdeffect ?

Reply:
I don't need to launch a coroutine here